docs/INSTALLATION.md

```diff
@@ -1,65 +1,82 @@
 # Installation
 
-## Installing Stack
 
-If you don't have a recent Stack version installed in your system, follow the
-[installation instructions](http://docs.haskellstack.org/en/stable/install_and_upgrade/)
-for your operating system. After completing the installation, test that you
-have installed it correctly by running this command in a terminal:
+## Get GHCup
 
-```bash
-stack --version
-```
+The recommended way of installing Haskell tools is through [GHCup][ghcup].
 
-If it outputs a Stack version equal to or greater than 1.1.2, then you have it
-installed correctly.
+With GHCup you can manage your installed compilers, project/package managers, and language server.
 
-## Installing GHC
 
-Stack will automatically download and install GHC (the Glasgow Haskell
-compiler) when it is required. You can start the installation and test
-by running the following command in a terminal:
+## Install Stack
 
-```bash
-stack ghc
-```
+For doing Exercism exercises you only need Stack.
+
+To install Stack, simply run the GHCup installation command or script and follow the defaults but specify that you would like to have Stack installed as well.
 
-If this outputs something like `ghc: no input files`, GHC is installed
-correctly.
+Did you miss the opportunity? No worries: just run `ghcup install stack` afterwards.
 
-Note that there are other ways to install GHC on a system (like the
-[Haskell platform](https://www.haskell.org/platform/)), but for
-running the Exercism tests, a regular stack installation is most
-likely to work correctly.
+GHCup will install
 
-## Installing HLint (optional)
+- **A compiler** (GHC: the Glasgow Haskell Compiler)
+  
+  Strictly speaking you do not need this for Exercism, but it comes in handy when you want to play with GHCi outside of exercise directories.
 
-HLint is a tool for suggesting possible improvements to Haskell code. These
-suggestions include ideas such as using alternative functions, simplifying
-code and spotting redundancies.
+- **Cabal**
 
-You can use Stack to install HLint:
+  A project/package manager. You do not need this for Exercism.
+
+- **Stack**
+
+  A project/package manager that you do need for Exercism. It will make sure you have the right compiler and libraries for each individual project (exercise).
+  
+To check that installation was successful, run the command `ghcup list` and/or `stack --version`.
```

Comment:
Is it self-evident what the student should be looking for as a result of the `ghcup list` and/or `stack --version` commands? If it is not, we will want to explain it here

Reply:
Throughout all of the instructions, we already implicitly assume that the student is familiar with the command line. So for `stack --version` I think we're good. As for `ghcup list`&hellip; see screenshot below. I do not know what failure might look like (other than the usual _command not found_).

<details><summary>Screenshot of <code>ghcup list</code></summary>

![image](https://user-images.githubusercontent.com/19817960/187233825-d9c70fd5-ae71-4291-9c7e-9e7f8384c0d1.png)
</details>

If you still think we need to be more explicit, I'll add explanations.

Reply:
It appears I need remember better what sort of background knowledge we assume for those who are joining a track. As you can see, that greatly affects what sort of detail we need to provide. In light of the assumption you've now informed me of, I now see that we can leave it as is.